Lib/test/test_dis.py

```diff
@@ -127,6 +127,21 @@ def _f(a):
        _f.__code__.co_firstlineno + 1,
        _f.__code__.co_firstlineno + 2)
 
+dis_f_with_positions = f"""\
+%-14s           RESUME                   0
+
+%-14s           LOAD_GLOBAL              1 (print + NULL)
+%-14s           LOAD_FAST                0 (a)
+%-14s           CALL                     1
+%-14s           POP_TOP
+
+%-14s           RETURN_CONST             1 (1)
+""" % tuple(map('%s:%s-%s:%s'.__mod__, [
+    tuple('?' if __p is None else __p for __p in __instruction.positions)
+    for __instruction in dis._get_instructions_bytes(
+        _f.__code__.co_code, co_positions=_f.__code__.co_positions(),
+    )
+]))
```

Comment:
I think this test doesn't cover the case where an instruction has no positions.

Also, it's not clear that it covers both valid position and '?'s.   The test function can assert that the positions include both of these cases.

Reply:
Thanks, there was indeed an issue! I tried to make something simple but I don't know how to fake some positions (I know how to remove them all but I don't know how to only fake a single position). Instead, I just created the `co_linetable` value manually... Is there any simpler alternative?